lib/preview.js

```diff
@@ -23,10 +23,14 @@ const {
   isUngatedForPreview,
 } = require('./preview/previewUtils');
 const { markRemoteFsDirty } = require('./preview/routes/meta');
-const { startShadowDevServer } = require('./preview/shadowDevServer');
+const { startSprocketMenuServer } = require('./preview/sprocketMenuServer');
```

Comment:
`shadowDevServer` -> `sprocketMenuServer` to be a little more obvious what it's for